h/api/nipsa/models.py

```diff
@@ -0,0 +1,30 @@
+import sqlalchemy
+from sqlalchemy.orm import exc
+
+from h import db
+
+
+class NipsaUser(db.Base):
```

Comment:
So as I mentioned in the document that we discussed yesterday, the API should probably have its own declarative base class.

I'm happy to overlook this for now, but if you do fancy having a stab at this it should be reasonably straightforward. You'd probably just need to add something like the following to `h/api/db.py`:

```python
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

def use_session(session, base=Base):
    base.query = session.query_property()

def bind_engine(engine, base=Base, should_create=False, should_drop=False):
    base.metadata.bind = engine
    if should_drop:
        base.metadata.drop_all(engine)
    if should_create:
        base.metadata.create_all(engine)
```

And then in `h/db.py#includeme` at an appropriate point add

```python
api.db.use_session(Session)
```

And then at the end of `h/db.py#bind_engine` add

```python
api.db.bind_engine(engine, should_create=should_create, should_drop=should_drop)
```

And hopefully that will give you a distinct Base class you can use, configured with a query property connected to the shared session.